django-server/server/admin/management/commands/generate_reviews.py

```diff
@@ -2,10 +2,98 @@
 from django.db import transaction
 
 from datetime import date, datetime, time, timedelta
-from random import uniform
+from random import uniform, choice
 
 from api.models import FixedMenuOption, FixedMenuOptionReview
 
+
+review_texts = [
+    "Magnificent!",
+    "Could've been worse",
+    "I enjoyed it",
+    "Salty",
+    "Loved the food, will come again.",
+    "Splendid, absolutely refined taste.",
+    "Kinda tasteless...",
+    "The taste was pretty lit if you'd ask me. The main dish's flavour was rockin'. Gonna definitely drop here more!",
+
+    "Delicious 😋",
+
+    "Ciekawy smak.",
+    "Pychota!",
+    "Bardzo dobre, polecam :)",
+    "Całkiem dobre",
+    "Średnie",
+    "Moja ulubiona opcja w menu",
+    "Jadłem lepsze",
+    "Polecam spróbować!",
+    "Niezłe jak na tę cenę.",
+    "Miodzio",
+    "Wiem kto mieszał ten rosół ;)",
+    "Smakowało :)",
+    "Niedobre",
+    "Nie było złe, ale nie urywa",
+    "Po pierwsze, co ja w ogóle zjadłem to ja nawet nie wiem. Koszmar! Ziemniaki niedogotowane,"  # continues
+    " mięso twarde! Jestem zawiedziony i chcę zwrotu.",
+
+    ":<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+    "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+    "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+    "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<",
+    "Bardzo dobre😂",
+    "😂😂😂😂😂",
+
+    "(╯°□°）╯︵ ┻━┻",
+    "あまい (´・ω・`)",
+    "甘かった！",
+    "まず！"
+
+    ".",
+]
+
+review_authors = [
+    "Stefan",
+    "Pawel",
+    "sebafor01",
+    "Czym_sa_monady?",
+    "Gabrysia",
+    "a.pestka",
+    "dark_michał",
+    "KaLorakkk",
+    "p4v31",
+    "1gor",
+    "david_504",
+    "xXx_Kucharz_xXx",
+    "Oleksandra",
+    "Robert'); DROP TABLE Camera_Event;--",
```

Comment:
:smile:

Reply:
that's not the name of table in our model. let's try with api_cameraevent xd